### PR TITLE
Repo-mapper web

### PR DESCRIPTION
**Pull request by patricksimonian**
## Repo Mapper Web

This is a simple web app that queries a volume inside the caddy pod for a csv file. 

## Notes

- the deployment config contains two containers:
   - the caddy container
   - the ansible container

### CODE REVIEW

File: tools/audit/repo-mapper/openshift/web.dc.yaml [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+            - containerPort: 9000
+              protocol: TCP
+          resources: {}
+          terminationMessagePath: /dev/termination-log
+          terminationMessagePolicy: File
+          volumeMounts:
+            - mountPath: /var/www/html/reports
+              name: output-dir
+        - image: ${NAME}-static:${VERSION}
+          name: repo-mapper
```
Comment (ShellyXueHan) [inline tools/audit/repo-mapper/openshift/web.dc.yaml:94]: parameterize the template, I've got the env var and name setup here: https://github.com/BCDevOps/platform-services/blob/master/tools/audit/repo-mapper/.openshift/deployment.yml

File: tools/audit/repo-mapper/openshift/web.dc.yaml [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+          cpu: '250m'
+          memory: '105Mi'
+    template:
+      metadata:
+        creationTimestamp: null
+        labels:
+          deploymentconfig: ${NAME}-static${SUFFIX}
+          app: ${NAME}-static${SUFFIX}
+      spec:
+        containers:
```
Comment (ShellyXueHan) [inline tools/audit/repo-mapper/openshift/web.dc.yaml:67]: missing service account

File: tools/audit/repo-mapper/openshift/web.dc.yaml (hunk 1 above)
Comment (ShellyXueHan) [reply tools/audit/repo-mapper/openshift/web.dc.yaml:94]: okay we are not using this dc. could be removed. `.openshift/web.deployment.yml` is used now.

File: tools/audit/repo-mapper/openshift/web.dc.yaml (hunk 2 above)
Comment (patricksimonian) [reply tools/audit/repo-mapper/openshift/web.dc.yaml:67]: missed this. those files are no longer need and have been deleted

File: tools/audit/repo-mapper/web/src/App.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+}
+
+const validateCb = shouldPoll => shouldPoll;
+
+const CSVLink = () =>  (
+  <div className="container">
+    <h2>
+      Report Complete!
+    </h2>
+    <a href="" download={process.env.REACT_APP_CSV_ROUTE}>Download</a>
```
Comment (ShellyXueHan) [inline tools/audit/repo-mapper/web/src/App.js:66]: does it work straight forward with the path? no need for anything like fs?

Comment (patricksimonian) [reply tools/audit/repo-mapper/web/src/App.js:66]: yes it does my baddd

**Pull request merged**